build.gradle

```diff
@@ -66,12 +66,12 @@ subprojects {
     }
 
     dependencies {
-        providedCompile group: 'org.eclipse.microprofile', name: 'microprofile', version: '2.2'
-        providedCompile group: 'javax', name: 'javaee-api', version: '8.0'
+        providedCompile group: 'org.eclipse.microprofile', name: 'microprofile', version: '3.0'
+        providedCompile group: 'jakarta.platform', name: 'jakarta.jakartaee-api', version: '8.0.0'
         testCompile group: 'junit', name: 'junit', version: '4.12'
-        testCompile group: 'org.eclipse', name: 'yasson', version: '1.0.3'
-        testCompile group: 'org.glassfish', name: 'jakarta.json', version: '1.1.5'
-        libertyRuntime group: 'io.openliberty', name: 'openliberty-runtime', version: '[19.0.0.8,)'
+        testCompile group: 'org.eclipse', name: 'yasson', version: '1.0.5'
+        //testCompile group: 'org.glassfish', name: 'jakarta.json', version: '1.1.5'
```

Comment:
Accidental comment?

Reply:
yea, I commented it out while I was testing its removal. I'll remove this commented out line and the other 2 occurrences in my DB PR that builds on top of this branch